zerobuf/data/frame.fbs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2016, Human Brain Project
+//                     Jafet.VillafrancaDiaz@epfl.ch
+//
+namespace zerobuf.data;
+
+// Event containing the data frame range information. It is based on the same
+// units as the zerobuf.render.Frame, i.e. number of frames.
+// It may be used as a base event that defines the available range of frames
+// coming from a certain data source, being also possible to use the
+// zerobuf.render.Frame event at the same time, which would define the frame
```

Comment:
This all could have been handled in the GUI, why did we add a new event ?
The GUI is the one who commands a frame ?